src/System.Management.Automation/FormatAndOutput/common/ComplexWriter.cs

```diff
@@ -699,13 +699,22 @@ internal static string StripNewLines (string s)
 #endif
         internal static string TruncateAtNewLine(string s)
         {
-            if (string.IsNullOrEmpty(s))
+            if (s == null)
```

Comment:
Why not:

```c#
if (string.IsNullOrEmpty(s))
{
  return string.Empty;
}
```

Reply:
I don't know whether .Net allocate `string.Empty` or `""` everytime, if don't allocate we should use `IsNullOrEmpty`.

Reply:
This pattern is so common I think it's safe to trust .Net to generate good code.

Reply:
Fixed.